src/alerting/components/__snapshots__/SearchBar.test.tsx.snap

```diff
@@ -0,0 +1,143 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This file is snapshot. It captures current approved state of the app, and notify developer about changes that need to be approved as new working version.

Reply:
Hmm I see. I don't think we want to test like that. I'd rather test for expected functionality instead of checking that the html matches some snapshot. Matching a snapshot doesn't tell us that the app is working _correctly_ - it just tells us when the app has changed. And if I'm opening a PR to change the Alerts SearchBar, I would already know that the component should change.

Reply:
I'd still like to remove this file before we merge this in please